frontend/components/base/Buttons/button.variants.ts

```diff
@@ -16,7 +16,8 @@ export const baseButtonVariants = tv({
         "bg-secondary text-secondary-foreground enabled:hover:bg-secondary/80",
       ghost:
         "bg-accent text-accent-foreground enabled:hover:bg-primary enabled:hover:text-primary-foreground",
-      link: "bg-transparent text-default-foreground underline underline-offset-4 enabled:hover:underline",
+      link: 
+        "bg-transparent text-default-foreground underline underline-offset-4 enabled:hover:underline pl-0 pr-0",
```

Comment:
Maybe because of a rebase but,

It should be `enabled:hover:no-underline` instead of `enabled:hover:underline`.

Reply:
fixed